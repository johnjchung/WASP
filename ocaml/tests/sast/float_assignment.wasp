Endpoint float_assignment(GET:) {
	float x = 3.;
	float y = 2.3;
}
