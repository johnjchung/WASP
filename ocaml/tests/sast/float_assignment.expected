Endpoint float_assignment( GET:) {float x = [float] 3.;, float y = [float] 2.3;}
